tests/paramgrill.c

```diff
@@ -831,13 +2572,20 @@ int main(int argc, const char** argv)
         filenamesStart=0,
         result;
     const char* exename=argv[0];
-    const char* input_filename=0;
+    const char* input_filename = 0;
```

Comment:
nit: prefer `NULL` instead of `0` for pointers.
It's _generally_ equivalent, though sometimes not : 
http://c-faq.com/null/machexamp.html